### DISCUSSION TITLE
Release/Tag outage?

### DISCUSSION THREAD

**Discussion by joshtwc** [community/community#110215] [Repositories]
### Select Topic Area

Bug

### Body

Posting here because I don't see any other information about this. 
I am trying to publish a release and it does not appear at the top of the releases "Latest" although the associated tag is at the top of the list. We pull our releases into wordpress by detecting updates however since it is not visible at the top of the list wordpress does not recognize that there is an update.
Has anyone else been experiencing this issue?

**Comment by pdefreitas** [comment]
Same for pushing commits to remote.

**Comment by joshtwc** [comment]
Looks like [GitHub Status](https://www.githubstatus.com/) has updated a couple services to Incident/Degraded

**Comment by joshtwc** [comment]
Appears to be working now.
